lib/actions/user.js

```diff
@@ -73,10 +73,12 @@ export function fetchOrInitializeUser (auth0) {
         // TODO: Improve AWS response.
 
         const isNewAccount = fetchStatus === 'error' || (user && user.result === 'ERR')
+        const userData = isNewAccount ? createNewUser(authUser) : user
+        dispatch(setCurrentUser({ accessToken, user: userData }))
+
+        // Also (asynchronously) load monitored trips for existing users.
```

Comment:
I don't think it needs to be stated that a fetch is asynchronous. I find the clarification more confusing than helpful.

Reply:
Updated in https://github.com/opentripplanner/otp-react-redux/pull/233/commits/9b74a576f40dd13476fa8c0f4a218dfe55dd6dd5